src/Symfony/Component/Serializer/Normalizer/DateTimeNormalizer.php

```diff
@@ -23,18 +23,26 @@
 class DateTimeNormalizer implements NormalizerInterface, DenormalizerInterface
 {
     const FORMAT_KEY = 'datetime_format';
+    const TIMEZONE_KEY = 'datetime_timezone';
 
     /**
      * @var string
      */
     private $format;
 
     /**
-     * @param string $format
+     * @var \DateTimeZone
```

Comment:
Oh. I misread. I though you were talking about the private `DateTimeNormalizer::getTimezone()`. 
While I agree, I added it to be consistent with the existing docblock. If we want to remove it, shouldn't we make a dedicated PR on the lowest branch rather than here?